proxy/contracts/test/SafeMock.sol

```diff
@@ -54,7 +54,7 @@ contract SafeMock is OwnableUpgradeable, ISafeMock {
         "EIP712Domain(uint256 chainId,address verifyingContract)"
     );
 
-    constructor() {
+    constructor() initializer {
```

Comment:
Is it legal?:)

Reply:
I'm badass. I don't care.

In current version of openzeppelin's `Initializable` init function can be called only during deployment or upgrade. So `__Ownable_init` can't be called in a constructor if it's not initializer.